scripts/scripts.js

```diff
@@ -0,0 +1,93 @@
+const app = {};
+
+// asynchronous fetch of local JSON data
+app.getData = async () => {
+    await fetch("./resources/resume.json")
+    .then(response => response.json())
+    .then(data => {
+        // call helper function to display all user data
+        app.displayAllUserData(data);
+    })
+    .catch(error => console.error(error));
+}
```

Comment:
since `fetch` already returns a promise, you actually don't need async/await here.
```
app.getData = () => fetch("./resources/resume.json")
    .then(response => response.json())
    .then(app.displayAllUserData)
    .catch(error => console.error(error))
```